docs/ubuntu_install.rst

```diff
@@ -1,30 +1,31 @@
 Ubuntu Install Guide
 ====================
 
-The *Application Server* and the *Monitor Server* specifically require
-the 64-bit version of `Ubuntu Server 14.04.2 LTS (Trusty
-Tahr) <http://old-releases.ubuntu.com/releases/14.04.2/>`__. The image
-you want to get is named ``ubuntu-14.04.2-server-amd64.iso``. In order
-to verify the installation media, you should also download the files
-named ``SHA256SUMS`` and ``SHA256SUMS.gpg``.
+The *Admin Workstation*, running Tails, should be used to download and verify
+Ubuntu Server.  The *Application Server* and the *Monitor Server* specifically
+require the 64-bit version of `Ubuntu Server 14.04.2 LTS (Trusty Tahr)
+<http://old-releases.ubuntu.com/releases/14.04.2/>`__. The image you want to get
+is named ``ubuntu-14.04.2-server-amd64.iso``. In order to verify the
+installation media, you should also download the files named ``SHA256SUMS`` and
+``SHA256SUMS.gpg``.
 
 Verify the Ubuntu installation media
 ------------------------------------
 
-First you should make sure that the Ubuntu image you downloaded hasn't
-been modified by a malicious attacker by checking its integrity with
-cryptographic signatures and hashes — which might sound complex but it's
-relatively easy to do. Before you can verify the Ubuntu installation
-media, you will need to download the associated public key.
+First, you should verify the Ubuntu image you downloaded hasn't been modified by
+a malicious attacker or otherwise corrupted. We can do so by checking its
+integrity with cryptographic signatures and hashes. First, we will download
+*Ubuntu Image Signing Key* and verify its fingerprint. Fortunately, we can do
+this in one step; GPG will implicity verify that the fingerprint of the key
+received matches the one requested when the full fingerprint is passed to the
+:code:`--recv-key` command.
 
 ::
 
-    gpg --keyserver pool.sks-keyservers.net --recv-key C5986B4F1257FFA86632CBA746181433FBB75451
-    gpg --fingerprint C5986B4F1257FFA86632CBA746181433FBB75451
+    gpg --keyserver hkp://qdigse2yzvuglcix.onion --recv-key C5986B4F1257FFA86632CBA746181433FBB75451
 
-The Ubuntu CD Image Automatic Signing Key should have a fingerprint of
-"C598 6B4F 1257 FFA8 6632 CBA7 4618 1433 FBB7 5451". If the fingerprint
-does not match what you see here, please get in touch at
+
+If the fingerprint does not match, please get in touch at
```

Comment:
Fixed in https://github.com/fowlslegs/securedrop/commit/caca462e2b3fb764587d7d42c3e6d0971fb80981